source/particle/world.cc

```diff
@@ -459,37 +460,41 @@ namespace aspect
           ++p;
         }
 
-      const UpdateFlags update_flags = property_manager->get_needed_update_flags();
-
```

Comment:
This is the part of the change that caused #6101. We used to get the update flags as `UpdateFlags`, but now they are `EvaluationFlags` obtained via a function argument.